app/assets/javascripts/compute_resources/xenserver/cache_refresh.js

```diff
@@ -0,0 +1,20 @@
+function refreshCache(attribute_name, compute_resource_id, on_success) {
+    tfm.tools.showSpinner();
+    url = "/foreman_xen/cache/refresh";
+    data = {
+        type: attribute_name,
+        compute_resource_id: compute_resource_id
+    }
+    $.ajax({
+            type:'post',
+            url: url,
+            data: data,
+            complete: function(){
+                tfm.tools.hideSpinner();
+            },
+            error: function(){
+                $.jnotify(__("Error refreshing cache for " + attribute_name), 'error', true);
```

Comment:
i would suggest using the notify function itself, we plan on replacing jnotify but will keep the notify function for compatibility.